Cargo.toml

```diff
@@ -59,8 +59,12 @@ image = { version = "0.24", features = ["jpeg", "png"] }
 copypasta = { version = "0.10.0", default-features = false, features = ["wayland", "x11"] }
 
 [features]
-default = ["editor"]
-# TODO: this is only winit and the editor serde, there are other dependencies that still depend on 
+default = ["editor", "rfd-tokio"]
```

Comment:
why do we need `rfd-tokio` by default?

Reply:
also there has to be one runtime selected for `ashpd` (by default it's `async-std`)

Reply:
But it doesn't need to be default in Floem?

Reply:
There should be some kind of default

Reply:
The default should be sync API, so no tokio, no async-std

Reply:
There is no sync API for dbus

Reply:
> The XDG Desktop Portal backend is used when the `gtk3` feature is disabled with [default-features = false](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features), and `xdg-portal` is enabled instead. Either the `tokio` or `async-std` feature must be enabled.

Reply:
Got it. Sorry didn't see that bit.

Reply:
Also, in current state, it already is async API, because only `async-std` was implemented before. Since 0.13.0, `tokio` runtime is provided as well.

Reply:
> Got it. Sorry didn't see that bit.

No problem :)